### ISSUE TITLE
pts-7.6.0m2 mafft-1.4.0 installation failed, wrong checksum

### ISSUE THREAD

**Issue by FabioLolix** [phoronix-test-suite/phoronix-test-suite#211]
Checksum Failed: http://fire-salamander.co.uk/pts-tests/pyruvate_decarboxylase.fasta.bz2

```
    pts/mafft-1.4.0:
        Test Installation 39 of 58
        2 Files Needed [0.38 MB / 1 Minute]
        Downloading: mafft-6.864-without-extensions-src.tgz                                                 [0.37MB]
        Estimated Download Time: 1m ................................................................................
        Downloading: pyruvate_decarboxylase.fasta.bz2                                                       [0.01MB]
        Estimated Download Time: 1m ................................................................................
            Checksum Failed: http://fire-salamander.co.uk/pts-tests/pyruvate_decarboxylase.fasta.bz2
    Try downloading the file again (Y/n): y
        Downloading: pyruvate_decarboxylase.fasta.bz2                                                       [0.01MB]
        Estimated Download Time: 1m ................................................................................
            Checksum Failed: http://fire-salamander.co.uk/pts-tests/pyruvate_decarboxylase.fasta.bz2
    Try downloading the file again (Y/n): n
            Downloading of needed test files failed.

```

**Comment by TheUltraCode** [comment]
I ran into this problem too. The reason why you are getting that error is that that link ("http://fire-salamander.co.uk/pts-tests/pyruvate_decarboxylase.fasta.bz2") does not "exist" anymore, let alone just "fire-salamander.co.uk". You can try it yourself: Just enter the URLs into your own browser, and see your browser display a 500 "internal server error".

To solve this, you need to download that file from elsewhere and "give it" to PTS. I downloaded my from here ([https://github.com/BCLinux/auto-test/blob/master/download-cache/pyruvate_decarboxylase.fasta.bz2](url)). After downloading the huge file, move it to your "~/.phoronix-test-suite/download-cache" directory, re-run PTS, and you should be in business. I did these steps myself, and PTS then successfully compiled the test for me (although I have not tested the test ( :) ) myself yet).

**Comment by michaellarabel** [comment]
The mirror issue is resolved by using pts/mafft-1.4.2.
